src/Questionnaire/Questionnaire.js

```diff
@@ -1,6 +1,9 @@
 import React, { useEffect, useState } from 'react'
 import { useHistory } from 'react-router-dom'
 import { personalityStyleData } from '../data/personalityStyleData'
+import HeadlineUnderline from '../common/HeadlineUnderline'
+import Button from '../common/Button'
+import styled from 'styled-components'
```

Comment:
Again, maybe with the `/macro`, I would say, that you either import it in all your components or not at all.